js/addAthletes.js

```diff
@@ -0,0 +1,60 @@
+const athletes = [
+  {
+    img: '../images/person1.jpg',
```

Comment:
[Optional]
Images aren't loading on deployed version because of path errors. It will be better if you fix image paths.

```suggestion
    img: './images/person1.jpg',
```